package.json

```diff
@@ -31,6 +31,8 @@
     "bunyan": "^1.8.12",
     "bunyan-format": "^0.2.1",
     "cache-manager": "^3.1.0",
+    "chai": "^4.2.0",
```

Comment:
@VISHNUDAS-tunerlabs why chai & mocha added to this file? are we added any test case

Reply:
also present in master